src/Views/Form/Templates/Classic/resources/js/form.js

```diff
@@ -22,6 +22,7 @@ domIsReady(() => {
     setupDonationLevels();
     moveDefaultGatewayDataIntoActiveGatewaySection();
     isDonationSummaryEnabled() && moveDonationSummaryAfterDonationAmountSection();
+    isFeeRecoveryInstalled() && attachFeeEvents() && updateFeesAmount();
```

Comment:
So `attachFeeEvents` and `updateFeesAmount` are both dependent on `isFeeRecoveryInstalled`, but is `updateFeesAmount` dependent on `attachFeeEvents`? `attachFeeEvents` doesn’t return anything so it’s `undefined`. Does `updateFeesAmount` ever run when `attachFeeEvents` always short circuits the expression? I think we should make this more clear and use an `if` statement.

Reply:
It works, so I’m just going to merge this.